src/Microsoft.OData.Edm/Vocabularies/Annotations/EdmVocabularyAnnotation.cs

```diff
@@ -4,17 +4,20 @@
 // </copyright>
 //---------------------------------------------------------------------
 
+using System;
+
 namespace Microsoft.OData.Edm.Vocabularies
 {
     /// <summary>
     /// Represents an EDM annotation with an immediate value.
     /// </summary>
-    public class EdmVocabularyAnnotation : EdmElement, IEdmVocabularyAnnotation
+    public class EdmVocabularyAnnotation : EdmElement, IEdmVocabularyAnnotationWithDefault
     {
         private readonly IEdmVocabularyAnnotatable target;
         private readonly IEdmTerm term;
         private readonly string qualifier;
         private readonly IEdmExpression value;
+        private readonly bool usesDefault;

```

Comment:
I don't think we need a field here since it is `readonly` and `UsesDefault` only has a getter